pkg/controller/supportbundlecollection/controller.go

```diff
@@ -401,6 +402,12 @@ func (c *Controller) createInternalSupportBundleCollection(bundle *v1alpha1.Supp
 			Password: authentication.BasicAuthentication.Password,
 		}
 	}
+	// Validate the format of the public key here, to catch invalid keys early.
+	if bundle.Spec.FileServer.HostPublicKey != nil {
```

Comment:
this reminds me we do also have a `validate` function. maybe that part need updates too.

Reply:
Done, thanks for pointing this out

Reply:
well i was trying to say PacketCapture also have a validate function.. but glad this also applied to supportbundle

Reply:
For packetcapture, I don't think we need to add extra validation for now (I wouldn't add a validation webhook just for that, but maybe others will disagree?). If the provided key is invalid, the upload will fail, just like it would fail if authentication data was missing or incorrect.

Reply:
i mean we already have a `validatePacketCapture` function to check protocol before, we can simply add this check here.